queries.sql

```diff
@@ -1,3 +1,20 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT name FROM animals WHERE date_of_birth >= '2016-01-01' AND date_of_birth <= '2019-12-31';
+
+SELECT name FROM animals WHERE neutered = true AND escape_attempts < 3;
+
+SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
+
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+
+SELECT * FROM animals WHERE neutered = true;
+
+SELECT * FROM animals WHERE name != 'Gabumon';
+SELECT * FROM animals WHERE name NOT IN ('Gabumon');
```

Comment:
[OPTIONAL] Nice implementation of the requested query :+1:  you can also make it happen with the following query:
```SQL
SELECT * FROM animals WHERE name <> 'Gabumon';
```
And I think this option makes more sense since we are comparing single `string` instead of collection.

Reply:
Thank you for the suggestion